### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project Requirements

- You will find all the details of the Bookstore design in Zeplin.

- Login to Zeplin (hint: you will find the credentials in the sneak peek of the project).

- Open the project Bookstore CMS.

- Use the information detailed in Zeplin to style the website.

- Your final design should match Zeplin's design.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All linter checks have passed.
No merge conflicts

File: src/components/Books-Input.js [hunk 1]
```diff
@@ -34,13 +34,15 @@ const InputBooks = () => {
   return (
     <section>
       <form onSubmit={handleSubmit}>
+        <div className="last-line" />
+        <h3>ADD NEW BOOK</h3>
```
Comment (omar-labana) [inline src/components/Books-Input.js:38]: - [ ] Please make sure to correctly align this form horizontally.
Your styling
image

File: src/components/Books-Page.js [hunk 2]
```diff
@@ -15,12 +15,34 @@ const Books = ({ books }) => {
...
+            <div>
+              <p className="category">{book.category}</p>
+              <span className="title">{book.title}</span>
+              <p className="author">Jasmine Cresswell</p>
+              <span className="comment">Comment</span>
+              <span className="line" />
+              <button className="remove" onClick={() => dispatch(deleteBook(book.id))} type="button">
+                Remove
+              </button>
+              <span className="line" />
```
Comment (omar-labana) [inline src/components/Books-Page.js:28]: - [ ] Kindly make sure to correctly position each section according to the design.|
Reference Design
image
Your styling
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Ugboaja-Uchechi, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ugboaja-Uchechi** [comment]
Fixed all code reviews as best as I could.

File: src/components/Books-Page.js (hunk 2 above)
Comment (Davidosky007) [reply src/components/Books-Page.js:28]: * Hi, @Ugboaja-Uchechi like the previous reviewer pointed out your design isn't properly aligned, from the screenshot below all `book cards` should be on a straight line
* ## ORIGINAL DESIGN 
uogo
* ## YOURS 
* The add button, the input area should be on a straight line and should be on a straight line as the `book cards` above it 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Ugboaja-Uchechi 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ugboaja-Uchechi** [comment]
Built this project for a screen size of 1440px

**Review by Emmyn5600 — APPROVED** [review]
Hi @Ugboaja-Uchechi 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**